spec/requests/posts/publish_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe 'PATCH /posts/:id/publish' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.